target/linux/ramips/dts/mt7621_wavlink_wl-wn531a6.dts

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "wavlink,wl-wn531a6", "mediatek,mt7621-soc";
+	model = "Wavlink WL-WN531A6";
+
+	aliases {
+		led-boot = &led_status_red;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_red;
+		label-mac-device = &gmac0;
```

Comment:
So, this is the same as wifi 2.4g?

Reply:
No, it's different. The label shows the MAC address of the 2.4G adapter. I changed the `label-mac-device` to point to that adapter. I also added `mtd-mac-address` prop in the adapter node.